calico_versioned_docs/version-3.27/release-notes/index.mdx

```diff
@@ -97,10 +97,14 @@ For more information, see [Enable the eBPF data plane](../operations/ebpf/enabli
  - eBPF: handles IPv4 packets with IP options. XDP only does so in kernels 5.18+ it drops them in older kernels. [calico #7701](https://github.com/projectcalico/calico/pull/7701) (@tomastigera)
  - eBPF - only report errors setting /proc/sys/net/../accept_local etc. if the device is present [calico #7631](https://github.com/projectcalico/calico/pull/7631) (@tomastigera)
 
+## Known issues
+
+- Calico panics if kube-proxy or other components are using nftable mode
```

Comment:
```suggestion
- Calico panics if kube-proxy or other components are using native `nftables` rules instead of the `iptables-nft` compatibility shim.
```